presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -1003,6 +1003,21 @@ private void assertFileFormat(HiveStorageFormat storageFormat)
         }
     }
 
+    @Test
+    public void testNotEqualFilterWithNaNs()
+    {
+        try {
+            assertUpdate("CREATE TABLE test_nan (double_value DOUBLE, float_value REAL)");
```

Comment:
- CREATE TABLE query should be outside of the try block
- finally block doesn't need "IF EXISTS"